lib/alchemy/elements_finder.rb

```diff
@@ -54,7 +54,7 @@ def elements(page_version:)
     def find_elements(page_version)
       return Alchemy::ElementsRepository.none unless page_version
 
-      elements = Alchemy::ElementsRepository.new(page_version.elements.available)
+      elements = Alchemy::ElementsRepository.new(page_version.elements.select(&:public?))
```

Comment:
I am not sure about this change. The elements we pass to the repository should be loaded as fast as possible from the database. Since we always only want published elements (it is de facto a default scope) we should use the scope imo.

Reply:
Outdated